crd/nodeinfo/api/v1alpha1/nodeinfo.go

```diff
@@ -0,0 +1,43 @@
+//go:build !ignore_uncovered
+// +build !ignore_uncovered
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// Important: Run "make" to regenerate code after modifying this file
+
+// +kubebuilder:object:root=true
+
+// NodeInfo is the Schema for the NodeInfo API
+// +kubebuilder:resource:scope=Namespaced
+// +kubebuilder:resource:shortName=ni
+// +kubebuilder:resource:path=nodeinfo
+// +kubebuilder:printcolumn:name="VMUniqueID",type=string,priority=1,JSONPath=`.spec.vmUniqueID`
```

Comment:
since this is the only important field in this CRD, maybe it should always be printed?
```suggestion
// +kubebuilder:printcolumn:name="VMUniqueID",type=string,priority=0,JSONPath=`.spec.vmUniqueID`
```